transport/src/main/java/io/netty/channel/DefaultChannelPipeline.java

```diff
@@ -1368,7 +1258,7 @@ public void channelReadComplete(ChannelHandlerContext ctx) throws Exception { }
 
         private static final String HEAD_NAME = generateName0(HeadContext.class);
 
-        private final Unsafe unsafe;
+        protected final Unsafe unsafe;
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this "protected" modifier. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://garage.netty.io/sonarqube/coding_rules#rule_key=squid%3AS2156)